modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractAdaCodegen.java

```diff
@@ -52,7 +66,8 @@ public AbstractAdaCodegen() {
         /*
          * Reserved words.  Override this with reserved words specific to your language
          */
-        setReservedWordsLowerCase(
+        // TODO Are those case sensitive or insensitive
```

Comment:
could you please answer this question @stcarrez @micheleISEP?

Reply:
Ada is case insensitive.

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)